.storybook/main.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  stories: [
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)",
+  ],
```

Comment:
```suggestion
  stories: [{
    directory: "../stories",
    titlePrefix: "",
    files: "*.stories.*"
  }],
```

This will cause Storybook to autogenerate the structure based on the file system structure, so you can avoid manually specifying the `title` field in each story file.